.gitattributes

```diff
@@ -1,2 +1,5 @@
 # Auto detect text files and perform LF normalization
 * text=auto
+*.png filter=lfs diff=lfs merge=lfs -text
+*.jpg filter=lfs diff=lfs merge=lfs -text
+*.pdf filter=lfs diff=lfs merge=lfs -text
```

Comment:
Did you test this change in Github Pages? Rendering the images isn't working on my local machine so I'm curious if this will be the same case on GP?

Reply:
Did you have [git-lfs](https://git-lfs.github.com/) (lfs stands for large file storage) installed in your local machine? You just need the `git lfs install` part.

git-lfs is supported natively in github.

Let me know if images rendered well for you afterwards.